ui/src/test/java/edu/wpi/grip/ui/codegeneration/tools/PythonPipelineInterfacer.java

```diff
@@ -0,0 +1,390 @@
+    StringBuilder temp = new StringBuilder();
+    temp.append("for contour in pipe.output");
+    temp.append(num);
+    temp.append(":\n");
+    temp.append("        print (\'c\')\n"
+        + "        for point in contour:\n"
+        + "            print (point[0][0])\n"
+        + "            print (point[0][1])\n");
+    BufferedReader in = runProcess(temp.toString());
+
+    List<MatOfPoint> contours = new ArrayList();
+    String nextIn;
+    assertTrue("empty contours", (nextIn = in.readLine()) != null);
+    while (nextIn != null) {
+      if (nextIn.equals("c")) {
+        nextIn = in.readLine();
+        List<Point> points = new ArrayList<>();
+        MatOfPoint tmpMat = new MatOfPoint();
+        while (!(nextIn == null || nextIn.isEmpty() || nextIn.equals("c"))) {
+          points.add(new Point(Integer.valueOf(nextIn), Integer.valueOf(in.readLine())));
+          nextIn = in.readLine();
+        }
+        tmpMat.fromList(points);
+        contours.add(tmpMat);
+      } else {
+        nextIn = in.readLine();
+      }
+    }
+
+
+    return contours;
+  }
+
+  private Object getOutputSize(int num) throws IOException {
+    StringBuilder temp = new StringBuilder();
+    temp.append("print (int(pipe.output");
+    temp.append(num);
+    temp.append("[0]))\n");
+    temp.append("print (int(pipe.output");
+    temp.append(num);
+    temp.append("[1]))\n");
+    BufferedReader in = runProcess(temp.toString());
+    int val1 = new Integer(in.readLine()).intValue();
+    int val2 = new Integer(in.readLine()).intValue();
+    System.out.println("value is : " + val1);
+    return new Size(val1, val2);
+  }
+
+  private Object getOutputNum(int num) throws IOException {
+    StringBuilder temp = new StringBuilder();
+    temp.append("print (pipe.output");
+    temp.append(num);
+    temp.append(")\n");
+    BufferedReader in = runProcess(temp.toString());
+    String input = in.readLine();
+    if (!input.equals("None")) {
+      double val = new Double(input).doubleValue();
+      return val;
+    } else {
+      return null;
+    }
+  }
+
+  private BufferedReader runProcess(String temp) throws IOException {
+    out.write(str.toString() + temp);
+    System.out.print("code: " + str.toString() + temp);
+    out.close();
+    ProcessBuilder pb = new ProcessBuilder("python3", "testing.py");
+    Process p = pb.start();
+
+    BufferedReader stdError = new BufferedReader(new InputStreamReader(p.getErrorStream()));
+
+    String s = null;
+    System.out.println("errors:");
+    while ((s = stdError.readLine()) != null) {
+      System.out.println(s);
+    }
+    try {
+      out = new BufferedWriter(new FileWriter("testing.py"));
+    } catch (IOException e) {
+      e.printStackTrace();
+    }
+    return new BufferedReader(new InputStreamReader(p.getInputStream()));
+  }
+
+  /* (non-Javadoc)
+   * @see edu.wpi.grip.ui.codegeneration.tools.PipelineInterfacer#setSwitch(int, boolean)
+   */
```

Comment:
This is an Overridden method. Javadoc is not needed -- it is automatically added from the super class.